### PR TITLE
Compile ug-minor-fixes branch on ML cluster and hook mozilla repos for all submodules

### PR DESCRIPTION
**Pull request by abhi-agg**
- Added .compute file with the compilation instructions
 - Hooked mozilla repos for all of its submodules (non-recursively):
      examples, regression-tests, nccl, sentencepiece, fbgemm and
      simple-websocket-server
 - Compiles successfully on ML cluster

### CODE REVIEW

**Review by kdavis-mozilla — CHANGES_REQUESTED** [review]
Why are there changes in the examples directory?

**Comment by kdavis-mozilla** [comment]
Oh you're moving all the submodules over, thus the changes in examples, regression-tests...

**Comment by abhi-agg** [comment]
> Oh you're moving all the submodules over, thus the changes in examples, regression-tests...

Yes, I thought the change related to hooking mozilla repos for submodules also belonged in the same commit.

File: .compute [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    libtool \
+    libgoogle-perftools-dev \
+    libpcre3-dev \
+    libprotobuf-dev \
+    protobuf-compiler \
+    python3-pip \
+    unzip \
+    wget
+
+# Build the server
```
Comment (kdavis-mozilla) [inline .compute]: I don't think this comment is correct.

COMPILE_SERVER defaults to OFF and it's not set to ON here.

**Review by kdavis-mozilla — COMMENTED** [review]
Just one "nit". The comment in the .compute seems to be wrong. Or am I wrong?

File: .compute (hunk 1 above)
Comment (abhi-agg) [reply .compute]: Agreed. Changed it in the next commit.

**Review by kdavis-mozilla — APPROVED** [review]
LGTM

**Pull request merged**